lib/slack/real_time/concurrency/celluloid.rb

```diff
@@ -38,6 +38,8 @@ def run_loop
           rescue EOFError => e
             logger.debug("#{self.class}##{__method__}") { e }
             driver.emit(:close, WebSocket::Driver::CloseEvent.new(1001, 'server closed connection')) unless @closing
+          ensure
+            terminate
```

Comment:
I think it might be good to check for the exception. I tried the code in the console. When I did an exit I got the following error:

```
Celluloid::DeadActorError: actor already terminated
       	/Users/harishs/.rvm/gems/ruby-2.3.0/gems/celluloid-0.17.3/lib/celluloid/proxy/actor.rb:34:in `terminate!'
       	/Users/harishs/.rvm/gems/ruby-2.3.0/gems/celluloid-0.17.3/lib/celluloid/proxy/cell.rb:64:in `terminate!'
       	/Users/harishs/.rvm/gems/ruby-2.3.0/gems/celluloid-0.17.3/lib/celluloid.rb:328:in `terminate'
       	/Users/harishs/.rvm/gems/ruby-2.3.0/gems/slack-ruby-client-0.7.4/lib/slack/real_time/concurrency/celluloid.rb:42:in `ensure in run_loop'
       	/Users/harishs/.rvm/gems/ruby-2.3.0/gems/slack-ruby-client-0.7.4/lib/slack/real_time/concurrency/celluloid.rb:42:in `run_loop'
```